docs/blog/pointblank-intro/index.qmd

```diff
@@ -0,0 +1,127 @@
+    .col_exists(columns=["date", "date_time"])
+    .interrogate()
+)
+
+validation
+```
+
+The table is chock full of the information you need when doing data validation tasks. And it's also easy on the eyes. Some cool features include:
+
+1. a header with information on the type of input table plus important validation options
+2. vertical color strips on the left side to indicate overall status of the rows
+3. icons in several columns (space saving and they let you know what's up)
+4. 'CSV' buttons that, when clicked, provide you with a CSV file
+5. a footer with timing information for the analysis
+
+It's a nice table and it scales nicely to the large variety of validation types and options available in the Pointblank library. Viewing this table is a central part of using that library and the great thing about the reporting being a table like this is that it can be shared by placing it in a publication environment of your choosing (for example, it could be put in a Quarto document).
+
+Here is the code that was used to generate the data validation above:
+
+```{python}
+#| eval: false
+
+import pointblank as pb
+
+validation = (
+    pb.Validate(
+        data=pb.load_dataset(dataset="small_table", tbl_type="polars"),
+        label="An example validation",
+        thresholds=(0.1, 0.2, 0.5),
+    )
+    .col_vals_gt(columns="d", value=1000)
+    .col_vals_le(columns="c", value=5)
+    .col_exists(columns=["date", "date_time"])
+    .interrogate()
+)
+
+validation
+```
+
+Pointblank makes it easy to get started by giving you a simple entry point (`Validate()`), allowing you to define as many validation steps as needed.
+
+Pointblank enables you to validate many types of DataFrames and SQL databases. Pointblank supports Pandas and Polars through Narwhals, and numerous backends (like DuckDB and MySQL) are also supported though our Ibis integration.
+
+### Exploring data validation failures
+
+Note that the above validation showed 6 failures in the first step. You might want to know exactly *what* failed, giving you a chance to fix the underlying data quality issues. To do that, you can use the `get_step_report()` method:
+
+```{python}
+validation.get_step_report(i=1)
+```
+
+The use of a table for reporting is ideal here! The main features of this step report table include:
+
+1. a header with summarized information
+2. the selected rows that contain the failures
+3. a highlighted column of interest
+
+Different types of validation methods will have step report tables that organize the pertinent information in a way that makes sense for the validation performed.
+
+### Previewing datasets across backends
+
+Because Pointblank supports many backends, with varying ways for displaying the underlying data, we provide the `preview()` function. With that you can get a beautiful and consistent view of any data table. Here is how it looks against a 2,000 row DuckDB table that's included in the package (`game_revenue`):
```

Comment:
Tweaked a bit to clarify that backends vary (not pointblank), which motivates `preview()`

```suggestion
Because many of the backends Pointblank supports have varying ways to view the underlying data, we provide a unified `preview()` function. It gives you a beautiful and consistent view of any data table. Here is how it looks against a 2,000 row DuckDB table that's included in the package (`game_revenue`):
```

Reply:
Nice! Definitely adding this in.